tasks.py

```diff
@@ -43,8 +43,16 @@ def test(c, verbose=False, keyword="", cov_lines=False):
 
 
 @task
-def doc(c):
+def build(c):
```

Comment:
```suggestion
def docbuild(c):
```
to not be mistaken with the package `build`